roles/calico_master/templates/calico.yml.j2

```diff
@@ -0,0 +1,400 @@
+---
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: calico-kube-controllers
+  namespace: kube-system
+rules:
+  - apiGroups:
```

Comment:
We should probably fix this in our upstream manifests - we're giving more permissions than we need here.

e.g. `extensions/pods` isn't required, nor is `""/networkpolicies"`